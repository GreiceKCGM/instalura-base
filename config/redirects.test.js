import redirects from './redirects';

describe('config/redirects', () => {
  test('renders all current redirect', () => {
    expect(redirects).toMatchSnapshot(); // Fotografia
  });
});
