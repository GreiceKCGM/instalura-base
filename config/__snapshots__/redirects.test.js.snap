// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`config/redirects renders all current redirect 1`] = `
Array [
  Object {
    "destination": "/app/login/",
    "permanent": true,
    "source": "/login/",
  },
]
`;
